setup.cfg

```diff
@@ -128,3 +128,39 @@ exclude_lines =
     def __repr__
     def __str__
     def _ipython_display_
+
+[mypy]
+python_version = 3.9
+
+files=weldx
+ignore_missing_imports = True
+strict_optional = False
+pretty = True
+allow_redefinition = True
+show_error_context = True
+show_error_codes = True
+show_column_numbers = True
+
+[mypy-weldx.asdf.*,weldx.tests.*]
+ignore_errors = True
+
+# fix current errors, remove module from ignore list, repeat
+[mypy-weldx.visualization.*]
```

Comment:
and fixing these is a week of work, isn't it?

Reply:
No